austation/code/modules/mob/living/carbon/human/human_defines.dm

```diff
@@ -2,3 +2,4 @@
 	var/ring_type = RING_DISABLED
 	var/ring_engraved = ""
 	var/ringsoff = FALSE
+	var/surrender_time = 0
```

Comment:
There's a better way to do this with timers and their ID's but this is fine for now if you don't want to mess with that stuff, it's certainly more deserving of memory than 3 rings variables 😆

Reply:
Me computer is broken, she’ll be right